### ISSUE TITLE
New GPU runtime: ROCm

### ISSUE THREAD

**Issue by louisfd** [tracel-ai/cubecl#28]
Make a ROCm runtime to target AMD

**Comment by saihaze** [comment]
Now that cubecl already has wgpu backend that can run on AMD GPU, is it any good to implement ROCm support?

**Comment by nathanielsimard** [comment]
Yeah, since you don't have access to lower precision types like f16, bf16 or even f8. Also wgpu doesn't expose instructions to program tensor cores equivalent for AMD.

**Comment by saihaze** [comment]
> Yeah, since you don't have access to lower precision types like f16, bf16 or even f8. Also wgpu doesn't expose instructions to program tensor cores equivalent for AMD.

Oh, that's really great! Thank you for your extrodinary work!
